src/dct.jl

```diff
@@ -171,3 +171,5 @@ end
     mul!(Array{T}(undef, p.plan.osz), p, copy(x)) # need copy to preserve input
 
 *(p::DCTPlan{T,K,true}, x::StridedArray{T}) where {T,K} = mul!(x, p, x)
+
+AbstractFFTs.AdjointStyle(::DCTPlan) = AbstractFFTs.UnitaryAdjointStyle()
```

Comment:
What will we do for the rest of the r2r plans?  e.g. the transpose of REDFT01 is REDFT10 + diagonal scaling.

Reply:
we're leaving r2r plans for a future PR. it can get pretty complicated with fishing multiple kinds of plans

Reply:
Specifically, we can define new adjoint styles here for those cases -- that's planned for a separate PR 🙂